config/printer-anycubic-vyper-2021.cfg

```diff
@@ -141,3 +169,139 @@ gcode:
     G4 P300
     SET_PIN PIN=probe_reset_pin VALUE=1
     G4 P100
+
+[virtual_sdcard]
+path: ~/gcode_files
+
+[pause_resume]
+
+
+#Klipper required macros
+[gcode_macro PAUSE]
+description: Pause the actual running print
+rename_existing: PAUSE_BASE
+gcode:
+    {% set e = params.E|default(10) %}        #edit to your retract length
+    PAUSE_BASE
+    {% if printer.extruder.can_extrude|lower == 'true' %}
+      G1 E-{e} F2100
+    {% else %}
+      {action_respond_info("Extruder not hot enough")}
+    {% endif %}
+    END_POS
+    M140 S55
```

Comment:
Not required. Optional. Not for base config files

Reply:
This is base on the current config file. I just add a function to the part of moving to park position. It raise the readibility.